Sources/LinguaLib/Infrastructure/LocalizationGenerator/Generator/PlatformLocalizationGenerator.swift

```diff
@@ -11,8 +11,12 @@ final class PlatformLocalizationGenerator: PlatformLocalizationGenerating {
   }
   
   func generateLocalizationFiles(data: [LocalizationSheet], config: Config.Localization) throws {
-    data.forEach { data in
-      try? localizedFileGenerator.generate(for: data, config: config)
+    try data.forEach { data in
+      do {
```

Comment:
Let's consider removing do catch block, as the same error is thrown already.

Reply:
Yes, not needed, redundant.